python/pynessie/cli.py

```diff
@@ -270,7 +270,7 @@ def log(  # noqa: C901
 
 def _format_log_result(x: CommitMeta) -> str:
     result = click.style("commit {}\n".format(x.hash_), fg="yellow")
-    result += click.style("Author: {} <{}>\n".format(x.committer, x.email))
+    result += click.style("Author: {}\n".format(x.committer))
```

Comment:
Oh, shouldn't this be `x.author`?

FYI: git doesn't show the committer, even in the "very verbose commit view mode".

Reply:
good catch, yes this should be `x.author`